system/scripts/core/calc_skill.cs

```diff
@@ -67,15 +67,15 @@ public float SCR_Get_SkillFactor(Skill skill)
 	public float SCR_Get_AbilityReinforceRate(Skill skill)
 	{
 		if (skill.Data.ReinforceAbility == 0)
-			return 0;
+			return 1;
```

Comment:
This change is incorrect. The result is used in this formula:
```cs
var value = sklFactor + (sklFactorByLevel * (skill.Level - 1));
// ...
value += value * byReinforceRate;
```
Meaning that if you return 1, the value would always double, because it's added to itself. We only want to return and add the bonus.

Reply:
I was looking into the SCR_REINFORCEABILITY_TOOLTIP from the client, I guess I was wrong then...